examples/run-blockchain/index.js

```diff
@@ -17,14 +17,24 @@ var blockchainDB = levelMem()
 
 var state = new Trie()
 
-var blockchain = new Blockchain(blockchainDB)
-blockchain.ethash.cacheDB = level('./.cachedb')
+var validate = false
```

Comment:
Not sure if this change is worth it. This is intended for new users of this project, who may be intrigued about this example not validating when it could.

Reply:
Yes, I can remove `validate`. I just copied from VM test case in order to make this example to work.

Reply:
done